### PR TITLE
Replicate all semaphore activities in ci and cd targets

### PR DESCRIPTION
**Pull request by deitch**
## Description

Replicate the workflow from kube-controllers PR here. It already had a `ci` target (which Semaphore already used), so just then `cd` side.

* Create a make cd target that only tags images and pushes to registries

As discussed with @caseydavenport @fasaxc @tomdee

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -307,6 +307,18 @@ test-install-cni: image k8s-install/scripts/install_cni.test
 ## Run what CI runs
 ci: clean static-checks test-cni-versions image test-install-cni
 
+## Deploys images to registry
```
Comment (caseydavenport) [inline Makefile]: @deitch I think we're going with these big block headings now - could you add one here for CD (I believe this is the convention @tomdee has been adding throughout)

Comment (deitch) [reply Makefile]: Sure. The double- comment `##` causes it to show up when you do `make help` (from [this line](https://github.com/deitch/calico-cni-plugin/blob/96659bfc872f2e38a32b9cf50af826aa347008f0/Makefile#L424) ) and happy to add the block heading too. Coming in one minute.

Comment (deitch) [reply Makefile]: Wait a minute, there is one here. See [this](https://github.com/deitch/calico-cni-plugin/blob/96659bfc872f2e38a32b9cf50af826aa347008f0/Makefile#L303-L305) . Is this not good enough?

Comment (tomdee) [reply Makefile]: Yep, I've just been adding /CD to that block heading (https://github.com/projectcalico/confd/blob/master/Makefile#L295) so lets go with that

Comment (deitch) [reply Makefile]: Done

**Pull request merged**